README.md

```diff
@@ -1 +1,111 @@
-# air-pollution-1
\ No newline at end of file
+<a id="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://user-images.githubusercontent.com/108806646/203941212-a33b97da-6d84-4649-b763-4e8191c24805.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [Air-Pollution](#air-pollution)
+  - [Built With](#built-with)
+  - [Live Demo](#live-demo)
+  - [Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Available Scripts](#available-scripts)
+  - [Author](#author)
+  - [Contributing](#contributing)
+  - [Acknowledgments](#acknowledgments)
+  - [License](#license)
+
+
+# Air-Pollution
+
+
+**Air-Pollution** This is a SPA mobile web application to check a list of metrics for the air quality of a specified area.
+
+## Built With
+
+<ul>
+  <li><a href="https://reactjs.org/">React.js</a></li>
+  <li><a href="https://www.ecma-international.org/">JavaScript</a></li>
+</ul>
+
+## Live Demo
+
+- [Live Demo Link](https://air-pollution-2ff568.netlify.app/)
+- [Video Link](https://www.loom.com/share/b506f44d82d14783bb31c97de930f698)
+
+
+## Getting Started
+
+> To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+  - A web browser _(Chrome, Mozilla, Edge...)_.
+  - A packet manager _(In the following instruction npm will be used, you can choose any packet manager you like)_
+
+### Setup
+
+1. Clone this project on your local machine using the following command on your Git Bash: `https://github.com/TechnoAndy/air-pollution-1`
+
+2. Browse to project directory `cd air-pollution-1`
+
+3. Run `npm install` To install project dependencies _(a packet manager is required)_.
+
+
+### Available Scripts
+
+In the project directory, you can run:
+
+- `npm start`
+
+  Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+  The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+- `npm test`
+
+  Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+- `npm run build`
+
+  Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+## Author
+
+👤 **Andrea Diegaardt**
+
+- GitHub: [@TechnoAndy](https://github.com/TechnoAndy)
+- Twitter: [ADiegaardt](https://twitter.com/ADiegaardt)
+- LinkedIn: [Andrea Diegaardt](https://www.linkedin.com/in/andy-diegaardt/)
+
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+## Acknowledgments
```

Comment:
Great job @TechnoAndy 🤝

- [ ] As stated in the project requirements, kindly provide acknowledgment to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Complete